<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="expressionEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="calculateButton">
        <property name="text">
         <string>Calculate...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="dataLabel">
        <property name="text">
         <string>Data:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="dataEdit">
        <property name="styleSheet">
         <string notr="true">background-color:rgb(255, 222, 255)</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="createAVariableButton">
          <property name="text">
           <string>Create a Variable</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteAVariableButton">
          <property name="text">
           <string>Delete a Variable</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="helpButton">
          <property name="text">
           <string>Help</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>468</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="variableEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="versionButton">
      <property name="text">
       <string>Version</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>725</width>
     <height>32</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile_F">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="actionOpen_O"/>
    <addaction name="actionSave_As_S"/>
   </widget>
   <addaction name="menuFile_F"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_O">
   <property name="text">
    <string>Open(&amp;O)</string>
   </property>
  </action>
  <action name="actionSave_As_S">
   <property name="text">
    <string>Save As...(&amp;S)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
